### PR TITLE
Created select component with tailwind

### PR DESCRIPTION
**Pull request by Oluwadaminiola**

### CODE REVIEW

File: web/ui/dashboard/tailwind.config.js [hunk 1]
```diff
@@ -57,7 +57,8 @@ module.exports = {
 				80: '#31323D',
 				60: '#5F5F68',
 				40: '#737A91',
-				20: '#E8E8E9'
+				20: '#E8E8E9',
+				10: '#EDEDF5'
```
Comment (horlah) [inline web/ui/dashboard/tailwind.config.js:61]: Where is this color from?

File: web/ui/dashboard/src/app/components/select/select.component.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	}
+
+  registerOnChange() {}
+
+	registerOnTouched() {}
+
+	writeValue() {}
+
+	setDisabledState() {}
+}
```
Comment (horlah) [inline web/ui/dashboard/src/app/components/select/select.component.ts:46]: Review indentation on this file

File: web/ui/dashboard/src/app/components/select/select.component.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+			hover:bg-primary-500
+			hover:border-grey-20
+			focus:border-primary-100
+			focus:bg-white-100
+			outline-none
+			rounded-4px
+			bg-[#F7F9FC]
+			py-12px
+			px-16px
+		"
```
Comment (horlah) [inline web/ui/dashboard/src/app/components/select/select.component.html:35]: Please test the different states of the styles here against the design system, hover, disabled and error state

**Review by horlah — CHANGES_REQUESTED** [review]

File: web/ui/dashboard/tailwind.config.js (hunk 1 above)
Comment (Oluwadaminiola) [reply web/ui/dashboard/tailwind.config.js:61]: this is the color used for almost all borders and also used for the required tags

**Review by horlah — APPROVED** [review]

**Pull request merged**